src/libmw/include/mw/crypto/Crypto.h

```diff
@@ -0,0 +1,89 @@
+#pragma once
+
+// Copyright (c) 2018-2019 David Burkett
+// Distributed under the MIT software license, see the accompanying
+// file LICENSE or http://www.opensource.org/licenses/mit-license.php.
+
+#include <mw/models/crypto/BigInteger.h>
+#include <mw/models/crypto/Commitment.h>
+#include <mw/models/crypto/RangeProof.h>
+#include <mw/models/crypto/BlindingFactor.h>
+#include <mw/models/crypto/Signature.h>
+#include <mw/models/crypto/ProofMessage.h>
+#include <mw/models/crypto/RewoundProof.h>
+#include <mw/models/crypto/Hash.h>
+#include <mw/models/crypto/PublicKey.h>
+#include <mw/models/crypto/SecretKey.h>
+#include <support/allocators/secure.h>
+
+#define CRYPTO_API
+
+//
+// Exported class that serves as a lightweight, easy-to-use wrapper for secp256k1-zkp and other crypto dependencies.
+//
+class CRYPTO_API Crypto
+{
+public:
+    //
+    // Creates a pedersen commitment from a value with a zero blinding factor.
+    //
+    static Commitment CommitTransparent(const uint64_t value);
+
+    //
+    // Creates a pedersen commitment from a value with the supplied blinding factor.
+    //
+    static Commitment CommitBlinded(
+        const uint64_t value,
+        const BlindingFactor& blindingFactor
+    );
+
+    //
+    // Adds the homomorphic pedersen commitments together.
+    //
+    static Commitment AddCommitments(
+        const std::vector<Commitment>& positive,
+        const std::vector<Commitment>& negative = { }
+    );
+
+    //
+    // Takes a vector of blinding factors and calculates an additional blinding value that adds to zero.
```

Comment:
Is this comment accurate?

Reply:
Nope, not at all. I think it used to describe adding inverse blinding factors, and then I later changed the function and butchered the comment.